bukkit/src/main/java/co/aikar/commands/BukkitLocales.java

```diff
@@ -48,6 +48,14 @@ public void loadLanguages() {
         addMessageBundles("acf-minecraft", pluginName, pluginName.toLowerCase());
     }
 
+    @Override
```

Comment:
You missed the Sponge and JDA versions, but I also would rather us not do it in this fashion.

The Locales class (in ACF) should maintain a list of class loaders, that's initially just its own, and then implementing classes should add their additional class loader IF its different than ACF's, so we can avoid scanning delays.

Reply:
I had forgotten Sponge and JDA existed, sorry about that. But yeah, it's a bit iffy having to manually register the ClassLoaders per-component of ACF. Maybe we should have some sort of way to just register/add the ClassLoaders then ACF will also scan those environments only IF it has yet to find a ResourceBundle which matches the name in its own ClassLoader environment.